app-ios/Sources/SessionFeature/SessionView.swift

```diff
@@ -43,10 +161,9 @@ struct SessionView_Previews: PreviewProvider {
         SessionView(
             store: .init(
                 initialState: .init(
-                    timetableItem: Timetable.companion.fake().timetableItems.first!
-                ),
+                    timetableItemWithFavorite: TimetableItemWithFavorite(timetableItem: Timetable.companion.fake().timetableItems.first!, isFavorited: false)),
```

Comment:
I think `fake()` should be useful 🙏🏼 
```suggestion
                    timetableItemWithFavorite: TimetableItemWithFavorite.companion.fake(),
```

Reply:
Fixed and added previews for other Views also in https://github.com/DroidKaigi/conference-app-2022/commit/d29010c8d34f65941cda35e01a00b7fcca4fcef3